apollo-sample/src/main/java/com/example/apollographql/sample/SampleApplication.java

```diff
@@ -15,10 +16,11 @@
 import retrofit2.converter.moshi.MoshiConverterFactory;
 
 public class SampleApplication extends Application {
-  private static final String BASE_URL = "https://githunt-api.herokuapp.com";
+  // Change localhost to your machine's local IP address when running from a device
+  private static final String BASE_URL = "http://10.6.1.232:8080";
```

Comment:
Agree with Martin. This should be a well known server. Everything that can be done to make it easy on someone looking into this for the first time should be considered right?